CommonTools/TrackerMap/src/TrackerMap.cc

```diff
@@ -1481,9 +1481,9 @@ void TrackerMap::save_as_fectrackermap(bool print_total,float minval, float maxv
         col =gROOT->GetColor(ncolor+100);
         if(col)
           col->SetRGB((Double_t)(red/255.),(Double_t)(green/255.),(Double_t)(blue/255.));
-        else
+        else {
           c = new TColor(ncolor+100,(Double_t)(red/255.),(Double_t)(green/255.),(Double_t)(blue/255.));
-          vc.push_back(c);
```

Comment:
Without the curly braces this pushes back a pointer to the last TColor set instead of the new one just created.